sample/src/main/java/com/pedrogomez/renderers/sample/model/RandomVideoCollectionGenerator.java

```diff
@@ -122,6 +124,11 @@ private void configureTitleAndThumbnail(final Video video) {
     video.setThumbnail(thumbnail);
   }
 
+  private void configureUniqueId(final Video video) {
+    int randomIndex = random.nextInt();
```

Comment:
You're using a collection with duplicated items so we need to provide a unique id per item in order to allow multi-selection correctly. In the real world, this is not needed because the items are usually different.